FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/statemachine/IntakeAutonomous.java

```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team1076.robot.statemachine;
+
+public class IntakeAutonomous extends AutoState {
+	private long duration, startTime;
+	private double intakeSpeed;
+
+	public IntakeAutonomous(int durationMilliseconds, double intakeSpeed) {
+		this.duration = durationMilliseconds * 1000 * 1000;
```

Comment:
This should use the time conversion API if you want to convert milliseconds to nanoseconds.